plane/source/docs/automatic-tuning-with-autotune.rst

```diff
@@ -82,61 +87,32 @@ When you engage AUTOTUNE mode a few things will happen:
    rates and angle error to demanded rate gain. These values depend on the :ref:`AUTOTUNE_LEVEL<AUTOTUNE_LEVEL>` 
 -  the autotune system will monitor your demanded roll and pitch rates
    (as determined by your transmitter stick movements). When the
-   demanded roll or pitch rate exceeds 80% of the maximum rate (not 80% stick deflection, one must move stick quickly towards full throw) the
-   autotune system will use the response of the aircraft to learn roll
-   or pitch tuning values.
--  the autotune system will save the parameters once the tune has been completed for an axis. This means that if autotune causes your aircraft to
-   become unstable switch to another mode and
-   recover. When you switch out of AUTOTUNE mode the last saved
-   parameters are restored. Switching back into AUTOTUNE will resume the tune where it was stopped, unless a reboot occurs. Note that the SMAX params may hide an over-tuned condition, such
-   that oscillations may appear as just "twitchy" behavior.
+   demanded roll or pitch rate (or yaw rate if :ref:`YAW_RATE_ENABLE<YAW_RATE_ENABLE>` = 1) exceeds 40% of the maximum tuning target axis rate set by the ref:`AUTOTUNE_LEVEL<AUTOTUNE_LEVEL>`,(not stick deflection) the autotune system will use the response of the aircraft to learn the tuning values. So sharp stick movements are required with a slight pause at moderate to maximum stick deflections in each direction.
+-  the autotune system will save the P and D parameters once the tune has been completed for an axis.  When you switch out of AUTOTUNE mode the FF and I values will be saved. Switching back into AUTOTUNE will resume the tune where it was stopped, unless a reboot occurs. 
```

Comment:
"unless a reboot occurs" -> will autotune resume if the systems is disarmed and armed again without a reboot? Don't laugh, this is a real use case with my rocket gliders that can have very short flights.

Reply:
I really should rework that a bit....once you have completed an axis P and D are stored until changed again...FF gets stored on mode exit....reboot,disarm, arm have no impact....leaving autotune in the middle of an axis reverts it to entry values...

Reply:
Thank you. Sorry to keep pushing on you here :=) I appreciate your patience with wording it just so.